docs/optimization_guide/ptq_introduction.md

```diff
@@ -29,7 +29,12 @@ To apply post-training methods in OpenVINO, you need:
 
 Currently, OpenVINO provides two workflows with post-training quantization capabilities:
 
-* :doc:`Post-training Quantization with POT <pot_introduction>` - works with models in OpenVINO Intermediate Representation (IR) only.
-* :doc:`Post-training Quantization with NNCF <nncf_ptq_introduction>` - cross-framework solution for model optimization that provides a new simple API for post-training quantization.
+* :doc:`Post-training Quantization with NNCF <nncf_ptq_introduction>` - cross-framework solution for model optimization that provides a simple API for post-training quantization.
+* :doc:`(Deprecated) Post-training Quantization with POT <pot_introduction>` - works with models in OpenVINO Intermediate Representation (IR) only.
```

Comment:
Do we still want to mention POT here? It's deprecated. We should still have the "how to use POT" docs for existing users, but IMO this document can focus on NNCF, and doesn't have to say that we have "two workflows" anymore.

Reply:
removed as I've changed the structure